google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseTest.java

```diff
@@ -67,6 +67,10 @@ public class DatabaseTest {
   private static final String DEFAULT_LEADER = "default-leader";
   private static final DatabaseDialect DEFAULT_DIALECT = DatabaseDialect.GOOGLE_STANDARD_SQL;
 
+  private static final boolean DEFAULT_DROP_PROTECTION = true;
```

Comment:
Default is false? The name is slightly misleading. Can we rename it to DROP_PROTECTION_ENABLED?